services/slam/builtin/builtin.go

```diff
@@ -664,7 +664,7 @@ func (slamSvc *builtIn) GetSLAMProcessConfig() pexec.ProcessConfig {
 	args = append(args, "-data_dir="+slamSvc.dataDirectory)
 	args = append(args, "-input_file_pattern="+slamSvc.inputFilePattern)
 	args = append(args, "-port="+slamSvc.port)
-	args = append(args, "--aix-auto-update")
+	args = append(args, "-aix-auto-update=1")
```

Comment:
This is necessary because google command line parsing complains if there's a parameter with no value.

Reply:
will this still run the appimage update for orbslam?

Reply:
I believe so, this is getting parsed as a bool: https://github.com/Otterverse/aix/blob/3241e360e756784743189a62131275d1a3c9037c/aix.go#L41

Reply:
Thank you for testing this and seeing it didn't work! I was able to fix this on the cartographer side.